src/pydantic_ome_ngff/latest/multiscales.py

```diff
@@ -72,27 +72,27 @@ class Multiscale(StrictVersionedBase):
     version: Optional[Any] = version
     # SPEC: why is this nullable instead of reserving the empty string
     # SPEC: untyped!
-    name: Optional[Any]
+    name: Optional[Any] = None
```

Comment:
This is an API change, I think: https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields . I think this maybe wants to have a default of `[]` rather than `None`, as the spec says it may not be present, but if present, it should be a list (rather than a `null`).

Reply:
`typing` actually has markers to deal with this: `NotRequired`. According to this discussion https://github.com/pydantic/pydantic/discussions/3563 it "will work as expected in V2", although I don't know whether I'd expect it to work only in the context of typeddicts or whether it will work more generally. No mention of it in the docs, but they haven't been fully updated yet.